### ISSUE TITLE
fluid跑ssd遇到这个错误，什么情况？

### ISSUE THREAD

**Issue by wenxingsen** [PaddlePaddle/models#1079]
python -u train.py --batch_size=64 --dataset='pascalvoc'  --pretrained_model='pretrained/pretrained/mobilenet_v1_imagenet.tar.gz'

-----------  Configuration Arguments -----------
ap_version: 11point
apply_distort: True
apply_expand: True
batch_size: 64
dataset: pascalvoc
is_toy: 0
learning_rate: 0.001
mean_value_B: 127.5
mean_value_G: 127.5
mean_value_R: 127.5
model_save_dir: model
nms_threshold: 0.45
num_passes: 120
parallel: True
pretrained_model: pretrained/pretrained/mobilenet_v1_imagenet.tar.gz
resize_h: 300
resize_w: 300
use_gpu: True
------------------------------------------------
/usr/local/lib/python2.7/dist-packages/paddle/fluid/evaluator.py:69: Warning: The DetectionMAP is deprecated, because maintain a modified program inside evaluator cause bug easily, please use fluid.metrics.DetectionMAP instead.
  % (self.__class__.__name__, self.__class__.__name__), Warning)
*** Aborted at 1532680007 (unix time) try "date -d @1532680007" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGSEGV (@0x0) received by PID 19513 (TID 0x7f5e97a11740) from PID 0; stack trace: ***
    @     0x7f5e9761a330 (unknown)
    @                0x0 (unknown)
Segmentation fault (core dumped)

**Comment by wenxingsen** [comment]
add_arg('parallel',         bool,  False,      "Parallel.")

把这个默认的ture改成false 跑通了，还在研究；

train on pascalvoc with 4701 images
test on pascalvoc with 311 images
Pass 0, batch 0, loss 40.2663459778, time 2.09887003899

**Comment by fangzong12** [comment]
你这个问题解决了么。我也出了这个错误

**Comment by fangzong12** [comment]
face_dectcion单个GPU可以跑通，多个GPU就报这个错误
*** Aborted at 1536888942 (unix time) try "date -d @1536888942" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGSEGV (@0x0) received by PID 4956 (TID 0x7f5667200700) from PID 0; stack trace: ***
    @     0x7f5666dfc390 (unknown)
    @                0x0 (unknown)
段错误 (核心已转储)

**Comment by wenxingsen** [comment]
> 
> 
> 你这个问题解决了么。我也出了这个错误

把那个parallel关掉就可以跑了

**Comment by fangzong12** [comment]
> > 你这个问题解决了么。我也出了这个错误
> 
> 把那个parallel关掉就可以跑了

不是说这个是多个GPU开关么

**Comment by qingqing01** [comment]
提供的错误信息太少， GPU并行需要安装nccl2。

**Comment by huihuiustc** [comment]
你们跑过几轮后遇到了loss值为nan的情况吗？
